src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -775,10 +775,10 @@ internal void UpdateRigiditySlider(float value, int mutationPoints)
         }
 
         rigiditySlider.Value = value;
-        SetRigiditySliderTooltip(value);
+        SetRigiditySliderTooltip(value / 10f);
```

Comment:
I feel like this change should be put into the `SetRigiditySliderTooltip` method, because this series of conversions doesn't make sense to me
editor (in float) -> comes to the GUI and is converted to int -> is converted back to float in order to display in the GUI